pyfolio/timeseries.py

```diff
@@ -584,6 +586,55 @@ def rolling_multifactor_beta(returns, df_multi_factor, rolling_window=63):
     return out
 
 
+def rolling_risk_factors(returns, risk_factors=None,
```

Comment:
I don't like calling this function "risk_factors" when it only computes Fama-French right now. We should have a function that can compute risk against any given factor, and this should be a specific case that calls that imho